.gitignore

```diff
@@ -0,0 +1,2 @@
+coverage/
```

Comment:
the gitignore file should look something like this

https://github.com/CraftAcademy/slowfood_api_team_3/blob/development/.gitignore

Reply:
that is fixed now